README.md

```diff
@@ -1,70 +1,176 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
+# Project Name
+<div align="center">
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+  <h2><b>Hello react frontend</b></h2>
+</div>
 
-## Available Scripts
+### [Link to back end](https://github.com/hammaazarok/hello-rails-back-end)
 
-In the project directory, you can run:
 
-### `npm start`
+<br>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# 📗 Table of Contents
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
```

Comment:
- [ ] Please edit your README file and kindly use the Microverse README template to make your README file more professional. and Kindly don't forget to add the link to the backend repo in it.